sdk/keyvault/keyvault-certificates/test/public/utils/testAuthentication.ts

```diff
@@ -49,7 +50,10 @@ export async function authenticate(
     throw new Error("Missing KEYVAULT_URI environment variable.");
   }
 
-  const client = new CertificateClient(keyVaultUrl, credential, { serviceVersion });
+  const client = new CertificateClient(keyVaultUrl, credential, {
+    serviceVersion,
+    httpClient: new DefaultHttpClient(),
```

Comment:
@xirzec this seems alright? I can't use `createXhrHttpClient` from `@azure/test-utils` since these are still on V1... not entirely sure how it all comes together but you have Identity on coreV2 (and I don't know which recorder), KV on coreV1 and old recorder... This works but is it correct?

Reply:
Actually doesn't seem to have fixed the issue 😢 going to look into it some more

Reply:
derp, I need to pass the XhrHttpClient from test-utils to the credential, not KV client - going to fix that up shortly